### ISSUE TITLE
How to save track.py results

### ISSUE THREAD

**Issue by hansoullee20** [mikel-brostrom/Yolov5_StrongSORT_OSNet#446]
### Search before asking

- [X] I have searched the Yolov5_StrongSORT_OSNet [issues](https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/issues) and found no similar bug report.

### Question

Hello
I've download the yolov5-crowdhuman.py model to track human only where the sources are in 416x416 image format. 
I have ran the following command:

python track.py --yolo-weights crowdhuman_yolov5m.pt --source /root/POC_dataset/dataset_processed/subsets/0000/val/images/ --classes 0 --name person

which then followed by the below texts on the terminal:

strong_sort/deep/reid/torchreid/metrics/rank.py:12: UserWarning: Cython evaluation (very fast so highly recommended) is unavailable, now use python evaluation.
  'Cython evaluation (very fast so highly recommended) is '
track: yolo_weights=['crowdhuman_yolov5m.pt'], strong_sort_weights=/root/Yolov5_StrongSORT_OSNet/weights/osnet_x0_25_msmt17.pt, config_strongsort=strong_sort/configs/strong_sort.yaml, source=/root/POC_dataset/dataset_processed/subsets/0000/val/images/, imgsz=[640, 640], conf_thres=0.5, iou_thres=0.5, max_det=1000, device=, show_vid=False, save_txt=False, save_conf=False, save_crop=False, save_vid=False, nosave=False, classes=[0], agnostic_nms=False, augment=False, visualize=False, update=False, project=runs/track, name=person, exist_ok=False, line_thickness=3, hide_labels=False, hide_conf=False, hide_class=False, half=False, dnn=False
Python 3.7.0 required by YOLOv5, but Python 3.6.9 is currently installed
YOLOv5 🚀 2022-7-4 Python-3.6.9 torch-1.10.1+cu102 CUDA:0 (NVIDIA TITAN RTX, 24220MiB)

Fusing layers... 
Model summary: 308 layers, 21041679 parameters, 0 gradients
Model: osnet_x0_25
- params: 203,568
- flops: 82,316,000
Successfully loaded pretrained weights from "/root/Yolov5_StrongSORT_OSNet/weights/osnet_x0_25_msmt17.pt"
** The following layers are discarded due to unmatched keys or layer size: ['classifier.weight', 'classifier.bias']
No detections
No detections
No detections
No detections
image 5/200 /root/POC_dataset/dataset_processed/subsets/0000/val/images/0006_0902_frame_2_1500_52.jpg: 640x640 1 person, Done. YOLO:(0.011s), StrongSORT:(0.013s)

However no results are saved. I see the new directories under runs/track directory, but the files appear to be empty.
How can I save the results preferably in image format?

Thank you for reading

**Comment by mikel-brostrom** [comment]
What do you want to save? Video results, MOT16 format compliant results?

Anyways, for video:

https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/c0f3a1f453134c1bf5e769cabdc55222d50b6ac6/track.py#L291

For  MOT16 / MOT17 txt results:

https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/c0f3a1f453134c1bf5e769cabdc55222d50b6ac6/track.py#L288

**Comment by remeberWei** [comment]
> 

hello, @mikel-brostrom, I used the method you provided but don't know why it doesn't work,And,I got it to work by adding the path parameter "--save-vid",as follows:
![image](https://user-images.githubusercontent.com/91881703/177731265-13312fa8-29da-403d-9f14-291790e329e7.png)

**Comment by mikel-brostrom** [comment]
For saving the video you have to add `--save-vid` to the `python track.py ...` command

**Comment by remeberWei** [comment]
yes，right. and i do not know how to train deepsort, can you help me? or give me some advices? Thank you!

**Comment by mikel-brostrom** [comment]
> or give me some advices?

Everything you need to know is here: [Deep appearance descriptor training (link to external repository)](https://kaiyangzhou.github.io/deep-person-reid/user_guide.html)

**Comment by remeberWei** [comment]
> > or give me some advices?
> 
> Everything you need to know is here: [Deep appearance descriptor training (link to external repository)](https://kaiyangzhou.github.io/deep-person-reid/user_guide.html)

Thank you！👍
